sparse/_coo/common.py

```diff
@@ -1059,6 +1059,102 @@ def clip(a, a_min=None, a_max=None, out=None):
     return a.clip(a_min, a_max)
 
 
+def unique_all(x, /):
+    """
+    Returns the unique elements of an input array x, the first occurring indices
```

Comment:
The docstrings can certainly be more complete.

Reply:
Right, I added `Parameters` and `Returns` sections.

Reply:
An examples section would be really nice too. 😄

Reply:
Right, done!